chart/compass/values.yaml

```diff
@@ -374,6 +374,8 @@ global:
     queueSize: 100
     # fetchTimeout - shows the timeout to wait for oauth token and for fetching systems (in one request) separately
     fetchTimeout: "5s"
+    # directorRequestTimeout - graphql requests timeout to director
+    directorRequestTimeout: "40s"
```

Comment:
Minor: Is `40s` something specific or just random? We usually set something that's a multiple of `30s`

Also maybe rename `fetchTimeout` to `fetchRequestTimeout` so that the configuration naming is consistent.

Reply:
`40s` is just a random number that will be sufficient for director requests - I think `30s` will be fine also.